src/managers/KeybindManager.cpp

```diff
@@ -964,6 +964,12 @@ void CKeybindManager::fullscreenActive(std::string args) {
     if (g_pCompositor->isWorkspaceSpecial(PWINDOW->m_iWorkspaceID))
         return;
 
+    PWINDOW->m_bDontSendFullscreen = false;
+    if (args == "2") {
+        PWINDOW->m_bDontSendFullscreen = true;
+        g_pCompositor->setWindowFullscreen(PWINDOW, !PWINDOW->m_bIsFullscreen, FULLSCREEN_FULL);
```

Comment:
why? why make another useless branch here? the other setWindowFullscreen will behave _exactly_ the same

Reply:
I removed the extra branch so it uses the other setWindowFullscreen